provisioning_templates/cloud_init/cloud_init_default.erb

```diff
@@ -24,10 +24,10 @@ hostname: <%= @host.name %>
 fqdn: <%= @host %>
 manage_etc_hosts: true
 users: {}
+<%= snippet 'redhat_register' %>
```

Comment:
This needs to be inside the
```
runcmd:
- |
```
block, which is specified below.

Reply:
Fixed